### PR TITLE
Fix multiple invocations of Quickstart

### LINKED ISSUE #900: Quick-start script fails if run a second time (with different data) without bringing containers down first
To reproduce:

The first run of the quick-start script succeeds.
```
$ wget https://raw.githubusercontent.com/Flowminder/FlowKit/master/quick_start.sh

$ bash quick_start.sh
Starting containers (this may take a few minutes)
Pulling flowdb                   ... done
Pulling flowdb_testdata          ... done
Pulling flowdb_synthetic_data    ... done
Pulling worked_examples          ... done
Pulling flowapi                  ... done
Pulling flowauth                 ... done
Pulling flowmachine_query_locker ... done
Pulling flowmachine              ... done
Pulling flowetl                  ... done
Pulling flowetl_db               ... done
[...]
All containers ready!
[...]
```

However, the second run fails if we try to bring up the system with `larger_data` but without running `bash quick_start.sh stop` first.
```
$ bash quick_start.sh larger_data
Starting containers (this may take a few minutes)
Existing containers are running and will be replaced. Are you sure? [y/N]? y

[...]
flowmachine_query_locker is up-to-date
Creating flowdb_synthetic_data ... 
flowauth is up-to-date
flowapi is up-to-date
Creating flowdb_synthetic_data ... error

ERROR: for flowdb_synthetic_data  Cannot start service flowdb_synthetic_data: driver failed programming external connectivity on endpoint flowdb_synthetic_data (174f969ed534104329099edb017829020356c81f62511bdfd6346185bf59300a): Bind for 0.0.0.0:9000 failed: port is already allocated

ERROR: for flowdb_synthetic_data  Cannot start service flowdb_synthetic_data: driver failed programming external connectivity on endpoint flowdb_synthetic_data (174f969ed534104329099edb017829020356c81f62511bdfd6346185bf59300a): Bind for 0.0.0.0:9000 failed: port is already allocated
ERROR: Encountered errors while bringing up the project.
```

### PR DESCRIPTION
**Pull request by greenape**
Closes #900

### I have:

- [ ] Formatted any Python files with [black](https://github.com/ambv/black)
- [x] Brought the branch up to date with master
- [x] Added any relevant Github labels
- [ ] Added tests for any new additions
- [ ] Added or updated any relevant documentation
- [ ] Added an Architectural Decision Record (ADR), if appropriate
- [x] Added an [MPLv2 License Header](https://www.mozilla.org/en-US/MPL/headers/) if appropriate
- [x] Updated the [Changelog](https://github.com/Flowminder/FlowKit/blob/master/CHANGELOG.md) 

### Description

- Simplifies the makefile a touch
- Makes all the compose files give FlowDB the same name, so that they work as overrides on the base compose
- Makes the Quickstart up recreate all anonymous volumes

Result being you can run Quickstart and happily switch between data volumes.

### CODE REVIEW

**Comment by greenape** [comment]
Did you set `GIT_REVISION=3161a84831f49cd31c5714927691b191201019e4`?

**Comment by jch1g10** [comment]
> Did you set `GIT_REVISION=3161a84831f49cd31c5714927691b191201019e4`?

Ah, no, I didn't. I'll try again.

**Comment by jch1g10** [comment]
> > Did you set `GIT_REVISION=3161a84831f49cd31c5714927691b191201019e4`?
> 
> Ah, no, I didn't. I'll try again.

Good, the quick-start script works fine when I remember to set `GIT_REVISION`.

File: Makefile [hunk 1]
```diff
@@ -13,160 +13,76 @@
...
-flowauth-build:
-	$(DOCKER_COMPOSE_BUILD) flowauth
-
-
-worked_examples-up: worked_examples-build
-	$(DOCKER_COMPOSE_UP) worked_examples
-
-worked_examples-down:
-	$(DOCKER_COMPOSE_DOWN) worked_examples
+	$(DOCKER_COMPOSE) down
```
Comment (jch1g10) [inline Makefile]: Previously  `make down` also removed volumes (i.e. `docker-compose down -v`). Is this a deliberate change?

File: Makefile [hunk 2]
```diff
@@ -13,160 +13,76 @@
...
 
-flowetl_db-up:
-	$(DOCKER_COMPOSE_UP) flowetl_db
+$(services:=-up):
+	$(DOCKER_COMPOSE) up -d $(subst _synthetic_data,,$(subst _testdata,,$(@:-up=)))
 
-flowetl_db-down:
-	$(DOCKER_COMPOSE_DOWN) flowetl_db
+$(services:=-down):
+	$(DOCKER_COMPOSE) down $(subst _synthetic_data,,$(subst _testdata,,$(@:-down=)))
```
Comment (jch1g10) [inline Makefile]: Previously `make <service>-down` did `docker-compose rm -f -s -v <service>`. Again, not sure if this is a deliberate change, but `docker-compose down <service>` is invalid usage so this target doesn't work at the moment.

File: Makefile [hunk 3]
```diff
@@ -13,160 +13,76 @@
...
-DOCKER_COMPOSE_BUILD = docker-compose -f $(DOCKER_COMPOSE_FILE) \
-	-f $(DOCKER_COMPOSE_SYNTHETICDATA_FILE) \
-	-f $(DOCKER_COMPOSE_TESTDATA_FILE) \
-	-f $(DOCKER_COMPOSE_FILE_BUILD) \
-	build
+DOCKER_COMPOSE_TESTDATA_FILE_BUILD ?= docker-compose-testdata-build.yml
+DOCKER_COMPOSE_SYNTHETICDATA_FILE_BUILD ?= docker-compose-syntheticdata-build.yml
+DOCKER_SERVICES ?= flowdb flowapi flowmachine flowauth flowmachine_query_locker flowetl flowetl_db worked_examples
+DOCKER_SERVICES_TO_START = $(patsubst flowdb%,flowdb,$(DOCKER_SERVICES))
+services := flowmachine flowapi flowauth flowdb worked_examples flowdb_testdata flowdb_synthetic_data flowetl flowetl_db
```
Comment (jch1g10) [inline Makefile]: Should `services` include `flowmachine_query_locker`?

**Pull request merged**